package.json

```diff
@@ -116,6 +116,7 @@
     "moment-timezone": "~0.5.33",
     "ng-chartist": "~4.1.0",
     "ng-mocks": "~13.5.2",
+    "ng2-charts": "~2.4.1",
```

Comment:
We already have d3 and probably other libraries for charts, what is benefit of this one?